apps/system/js/notifications.js

```diff
@@ -426,9 +430,9 @@ var NotificationScreen = {
     notificationNode.dataset.type = type;
     notificationNode.dataset.manifestURL = manifestURL;
 
-    if (detail.icon) {
+    if (detail.appIcon) {
```

Comment:
What is this change about? The string appIcon doesn't appear anywhere else in your patch, so it doesn't look like you're changing the code that generate these notification event details.  Was this a bug in the existing code?